control_toolbox/CMakeLists.txt

```
@@ -60,6 +66,15 @@ set(CONTROL_FILTERS_INCLUDE_DEPENDS
   tf2
   tf2_ros
 )
+set(CONTROL_FILTERS_INCLUDE_TARGETS
+  ${geometry_msgs_TARGETS}
```

```suggestion
  filters::filter_base
  ${geometry_msgs_TARGETS}
```
https://github.com/ros/filters/blob/ros2/CMakeLists.txt#L54